lib/timers.js

```diff
@@ -149,11 +160,26 @@ function setTimeout(callback, after, arg1, arg2, arg3) {
   return timeout;
 }
 
-setTimeout[customPromisify] = function(after, value) {
+setTimeout[customPromisify] = function(after, value, abortController) {
```

Comment:
Should this receive the `abortSignal` here instead? That’s my understanding of how the API is intended to be used, and it is also how e.g. `fetch()` uses it.

Reply:
I think generally we'll want to aim for this:

```suggestion
setTimeout[customPromisify] = function(after, value, { signal }) {
```

Reply:
Yep, that works.

Reply:
Updated to use the `{ signal }` form